src/components/Queries/QueriesDrawer.js

```diff
@@ -23,6 +23,7 @@ import { AiOutlineBars } from "react-icons/ai";
 import { queryMap } from "../../assets/data/queries";
 
 const QueriesDrawer = ({ usePredefinedQuery, displayText, setValue }) => {
+  
```

Comment:
```suggestion
```

Reply:
@janvi01 got all the points mentioned by you, I will fix it as per the demand